client/elements/text/sc-simple-text.html

```diff
@@ -384,91 +453,153 @@
           }
 
           _removeTooltip(v, tooltip, states, e) {
-              let timeToWait = e.sourceCapabilities.firesTouchEvents ? 10 : 100;
+              let timeToWait = this._getTooltipDelayTime(e);
               setTimeout(() => {
                   if (!states.isWordHovered && !states.isTooltipHovered) {
                       v.style.color = '';
                       v.removeAttribute('id');
                       tooltip.hide();
                       states.isTooltipShown = false;
+                      let lastElement = v;
+                      for(let i = 0; i < tooltip.elementCount; i++){
+                        let nextElement = lastElement.nextElementSibling;
+                        if(!nextElement){
+                            break;
+                        }
+                        lastElement = nextElement;
+                        if(!nextElement.classList.contains('lookup_element')){
+                            i--;
+                            continue;
+                        }
+                        if(!nextElement.id){
+                            nextElement.classList.remove('green-color');
+                        }
```

Comment:
the loop body could be a separate function

Reply:
no need for that.